node/test/max-call-overhead.js

```diff
@@ -99,18 +106,20 @@ allocCluster.test('request() with large arg1', {
                 'cn': 'wat'
             }
         }).send(arg1, 'a', 'b', onResponse);
-    });
+    }
 
     function onResponse(err, resp, arg2, arg3) {
-        assert.ok(err);
-        assert.equal(err.type, 'tchannel.arg1-over-length-limit');
-        assert.equal(err.message,
-            'arg1 length 16385 is larger than the limit 16384'
-        );
+        assert.fail("shouldn't get a response callback");
 
-        assert.equal(null, resp);
+        // assert.ok(err);
```

Comment:
Maybe, it's here in case we to the TODO above about making this a non-throwing path again.